shell/platform/tizen/tizen_view_nui.cc

```diff
@@ -74,6 +74,10 @@ void TizenViewNui::RequestRendering() {
   rendering_callback_->Trigger();
 }
 
+void TizenViewNui::Unfocus() {
+  FT_UNIMPLEMENTED();
```

Comment:
Is this difficult to implement?

Reply:
As you can see. nui implementation is quite different from elementary implementation.
I don't know much about this yet, and I have to study it. I need more time for this.

Reply:
Since NUI's TouchEvent Callback is located in flutter-tizen, it should be implemented with flutter-tizen(with additional APIs(?)). I will check after this patch is applied.

Reply:
I think we should finalize the design before applying this change.

Reply:
Ah, then, it seems that the `Unfocus()` method and the `focused` member should be implemented in `TizenView`.

Reply:
@JSUYA I moved them to `TIzenView`

Reply:
I made a PR applicable to NUIView. (https://github.com/flutter-tizen/engine/pull/354)
I changed `Unfocus()` to `SetFocus(bool f)` because there are cases where I need to make `focused_` true.